compute/admin_guide/authentication/oidc.adoc

```diff
@@ -116,12 +124,101 @@ Configure Prisma Cloud.
 
 .. Click *Save*.
 
+[#azure-ad]
+[.task]
+=== Azure Active Directory (AD)
+
+To integrate with Azure Active Directory (AD), you must register Prisma Cloud as an Open ID Connect (OIDC) application in Azure and configure Azure AD as an identity provider in Prisma Cloud.
+
+[.procedure]
+
+. Go to https://portal.azure.com/#home[your Azure console].
+
+. Find the Azure AD service.
+
+. Click the *app registration* button and select *New registration*
+
+. Enter a name and select *Accounts in this organizational directory only* as the supported account type.
+
+. Under *Redirect URI* select *Web console URL* and enter the path to the Prisma Cloud Console API for OIDC, for example *\https://<WEB_CONSOLE_URL>:8083/api/v1/authenticate/callback/oidc*
```

Comment:
We can't recommend the use of non-versioned APIs in our docs. Do we have an alternative for customers?

Reply:
@rcaballeromx I don't believe we do... @Tomerit are you familiar with any other way the customer can configure the integration without using this API (https://<WEB_CONSOLE_URL>:8083/api/v1/authenticate/callback/oidc)?

In any case, I'll add this API to the GH issue for adding supported APIs in Lagrange

Reply:
I'm not sure I follow why is this route problematic

Reply:
@Tomerit The issue is that the `authenticate/callback/oidc` API is not supported. You can review the versioned and supported options of the [API in our docs](https://prisma.pan.dev/api/cloud/cwpp/authenticate).
That's why @grevach is adding it to the GH issue to make it part of the versioned API.